libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2359,42 +2359,6 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     ],
     keywords: ["sol", "solana"],
   },
-  stakenet: {
-    type: "CryptoCurrency",
-    id: "stakenet",
-    coinType: 384,
-    name: "Stakenet",
-    managerAppName: "XSN",
-    ticker: "XSN",
-    scheme: "xsn",
-    color: "#141828",
-    supportsSegwit: true,
-    family: "bitcoin",
-    blockAvgTime: 60,
-    bitcoinLikeInfo: {
-      P2PKH: 76,
-      P2SH: 16,
-      XPUBVersion: 0x0488b21e,
-    },
-    units: [
-      {
-        name: "xsn",
-        code: "XSN",
-        magnitude: 8,
-      },
-      {
-        name: "satoshi",
-        code: "sat",
-        magnitude: 0,
-      },
-    ],
-    explorerViews: [
-      {
-        tx: "https://xsnexplorer.io/transactions/$hash",
-        address: "https://xsnexplorer.io/addresses/$address",
-      },
-    ],
-  },
```

Comment:
We don't want to totally remove it from currencies, because we just delist it from Live but the Nano app may still be supported on the manager.

Also, removing this will make Live crash at startup if the user already has Starknet accounts. Basically the `currency.id` field is very sensitive for all currencies here, as soon as any user has accounts with this id, its value can never change and it can never be removed from this list.

The right way to go seems to be to add a `terminated` link, see other examples in this file.